airflow/contrib/hooks/gcp_spanner_hook.py

```diff
@@ -196,7 +194,7 @@ def get_database(self, instance_id, database_id, project_id=None):
             database. If set to None or missing, the default project_id from the GCP connection is used.
         :type project_id: str
         :return: Database object or None if database does not exist
-        :rtype: Union[Database, None]
+        :rtype: Union[google.cloud.spanner_v1.database.Database, None]
```

Comment:
```suggestion
        :rtype: google.cloud.spanner_v1.database.Database or None
```

I think is the correct way of doing this in sphinx

Reply:
Fixed.